runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/ContactSection/ContactForm/ContactForm.tsx

```diff
@@ -89,8 +90,17 @@ const ContactForm: FC = () => {
             });
     };
 
+    const processingDataText = [...Array(7).keys()].map((el) => (
+        <li key={el}>
+            <Typography>
+                {/* @ts-ignore */}
+                {translate(`Landing.Contact.Form.ProcessingData.${el + 1}`)}
+            </Typography>
+        </li>
+    ));
```

Comment:
next time try to stick to convention (PascalCase divided by periods)
ProcessingData.${el + 1} -> ProcessingData.Point${el + 1}